CHANGELOG.md

```diff
@@ -6,12 +6,16 @@ IMPROVEMENTS:
 
 - commands/suspend: Introduce flag for suspending all machines [GH-9829]
 - commands/global-status: Improve message about removing stale entries [GH-9856]
+- provisioners/ansible_local: Improve installation from PPA on Ubuntu guests.
+    The compatibility is maintained only for active long-term support (LTS) versions,
+    i.e. Ubuntu 12.04 (Precise Pangolin) is no longer supported. [GH-9879]
```

Comment:
I don't think it is worth to put this into a BREAKING CHANGES section... what do you think?

Reply:
This seems fine to me 👍